llvm/tools/llvm-profgen/ProfileGenerator.cpp

```diff
@@ -76,13 +76,16 @@ static cl::opt<int, true> CSProfMaxContextDepth(
     cl::location(llvm::sampleprof::CSProfileGenerator::MaxContextDepth));
 
 static cl::opt<double> HotFunctionDensityThreshold(
-    "hot-function-density-threshold", llvm::cl::init(1000),
-    llvm::cl::desc(
-        "specify density threshold for hot functions (default: 1000)"),
+    "hot-function-density-threshold", llvm::cl::init(20),
```

Comment:
The switch name is also no longer accurate, since we are no longer selecting the top N functions. Perhaps `profile-density-threshold` is good enough.

Reply:
Can you share some date to show that 20 is a good threshold?